wifi.go

```diff
@@ -153,9 +153,12 @@ type StationInfo struct {
 	// The current data transmit bitrate, in bits/second.
 	TransmitBitrate int
 
-	// The signal strength of this station's connection, in dBm.
+	// The signal strength of the last received PPDU, in dBm.
 	Signal int
 
+	// The average signal strength, in dBm.
```

Comment:
Do we know over what time this is averaged?

Reply:
It is not specified in the uapi header, and it appears to be driver/firmware dependent from what I saw in the kernel code.

Reply:
Boo, oh well.